src/environments/environment.ts

```diff
@@ -1,21 +1,20 @@
 // This file can be replaced during build by using the `fileReplacements` array.
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
-import * as keys from './keys';
 
 export const environment = {
   production: false,
   timeTrackerApiUrl: 'http://localhost:7071/api',
   stackexchangeApiUrl: 'https://api.stackexchange.com',
 };
 
-export const AUTHORITY = keys.AUTHORITY;
-export const CLIENT_ID = keys.CLIENT_ID;
-export const SCOPES = keys.SCOPES;
+export const AUTHORITY = process.env.AUTHORITY;
```

Comment:
Why is using process.env.AUTHORITY, and not process.env["AUTHORITY"] ??
I think you should change that here, because from what I remember process.env.VARIABLE_NAME expects a .env file, but we are not using any files to hold the ENV variables anymore with these changes.